<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainDialog</class>
 <widget class="QDialog" name="TrainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>371</width>
     <height>321</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>员工甲                   一级员工</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/image/lisi.png</normaloff>:/image/lisi.png</iconset>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>员工乙                   二级员工</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/image/wujiu.png</normaloff>:/image/wujiu.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>员工丙                   一级员工</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/image/sunqi.png</normaloff>:/image/sunqi.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>员工丁                   三级员工</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/image/zhaoliu.png</normaloff>:/image/zhaoliu.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>员工戊                   高级员工</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/image/wangwu.png</normaloff>:/image/wangwu.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>员工己                   初级员工</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/image/zhangsan.png</normaloff>:/image/zhangsan.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>员工庚                   顶级员工</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/image/zhouba.png</normaloff>:/image/zhouba.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>员工辛                   二级员工</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/image/zhengshi.png</normaloff>:/image/zhengshi.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>员工壬                   一级员工</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/image/xiongda.png</normaloff>:/image/xiongda.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>员工葵                   高级员工</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/image/zhengshi.png</normaloff>:/image/zhengshi.png</iconset>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>10</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>增加人员</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/background_meitu_2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="RecordButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>培训记录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PlanButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>培训计划</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ReturnButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>AddButton</zorder>
  <zorder>lineEdit</zorder>
  <zorder>listWidget</zorder>
  <zorder>RecordButton</zorder>
  <zorder>PlanButton</zorder>
  <zorder>DeleteButton</zorder>
  <zorder>ReturnButton</zorder>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
